config/samples/kcp/dev_workspace/appstudio_v1alpha1_applicationsnapshot.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: appstudio.redhat.com/v1alpha1
```

Comment:
I guess this file should also be renamed, right?

Reply:
fixed